### ISSUE TITLE
Missing extension unaccent

### ISSUE THREAD

**Issue by carukc** [jkraemer/redmine_postgresql_search#1]
Hello,

Cool plugin.  Thank you for providing.  

I have tried to install but during migrate the script complains unaccent being unavailable. 

I've tried to enable in Postgres by logging into the database and running:

  CREATE EXTENSION unaccent;

but that does not seem to have resolved the problem.  Is there something else I need to do to get the plugin running?

Thanks
Chris

**Comment by jkraemer** [comment]
Strange, did you check the PostgreSQL logs? What version of PostgreSQL are you running on, and on what OS?

**Comment by carukc** [comment]
Hi… Thanks for the reply.

The log says:
2015-10-06 10:16:22 BST STATEMENT:  CREATE EXTENSION IF NOT EXISTS "unaccent"
2015-10-06 10:16:23 BST LOG:  could not receive data from client: Connection reset by peer
2015-10-06 10:16:52 BST ERROR:  permission denied to create extension "unaccent"

This is what told me that I needed to enable on Postgres.  Which I did using an account with full privileges.  My best guess is that it’s not enabled on all accounts when you enable once in Postgres (using a superuser account).  So… I’ve been trying to figure out how to adjust the permissions to allow this particular user to do more in this particular database.

I’m running Postgres 9.3 on Ubuntu 14.04… I was also going to upgrade to 9.4 to see if it makes a difference.
